### PR TITLE
Update cheatsheet.md

### PR DESCRIPTION
**Pull request by jayeshmahajan**
updating docs with new way to create pods.

### CODE REVIEW

**Comment by jayeshmahajan** [comment]
I signed it

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ kubectl apply -f ./my1.yaml -f ./my2.yaml      # create from multiple files
 kubectl apply -f ./dir                         # create resource(s) in all manifest files in dir
 kubectl apply -f https://git.io/vPieo          # create resource(s) from url
 kubectl create deployment nginx --image=nginx  # start a single instance of nginx
+kubectl run --generator=run-pod/v1 nginx --image=nginx # create nginx pod without creating deployment
```
Comment (sftim) [inline content/en/docs/reference/kubectl/cheatsheet.md]:
```suggestion
kubectl run --generator=run-pod/v1 nginx --image=nginx # create nginx pod without creating a Deployment
```

@jayeshmahajan you should also indent the comments on lines 93 to 99 to match the new line you're adding

**Comment by sftim** [comment]
I'm also aware that [generators are being removed from kubectl](https://github.com/kubernetes/kubernetes/pull/87077).

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-18110--kubernetes-io-master-staging.netlify.com/docs/reference/kubectl/cheatsheet/

File: content/en/docs/reference/kubectl/cheatsheet.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/reference/kubectl/cheatsheet.md]: Hello @jayeshmahajan .
In the kubectl cheatsheet, the comments use lowercase for resource names (deployment vs Deployment). The comment seems okay.
As @sftim mentioned, typically, when a new command is added, the comments are lined up for that section.
Would you have time to align the comments for the other commands to match your addition?
Thanks!

**Comment by kbhawkey** [comment]
Ping @jayeshmahajan

**Comment by jimangel** [comment]
@jayeshmahajan I am closing this issue since it has been open over 25 days with no response. Feel free to reopen after resolving conflicts / feedback. Thanks!

/close

**Pull request closed without merging**